<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="arunkumarpi-gift" ignored="0" name="login page test" tests="3" failures="1" timestamp="26 May 2021 13:48:12 GMT" time="18.983" errors="0">
  <testcase name="verifyLoginPageTitleTest" time="0.016" classname="com.boss.qa.tests.LoginPageTest">
    <skipped/>
  </testcase> <!-- verifyLoginPageTitleTest -->
  <testcase name="verifyLoginPageTitleTest" time="0.004" classname="com.boss.qa.tests.LoginPageTest">
    <failure type="java.lang.AssertionError" message="Login Page Title Not Matched expected [Login123 - KWI] but found [Login - KWI]">
      <![CDATA[java.lang.AssertionError: Login Page Title Not Matched expected [Login123 - KWI] but found [Login - KWI]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.boss.qa.tests.LoginPageTest.verifyLoginPageTitleTest(LoginPageTest.java:24)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- verifyLoginPageTitleTest -->
  <testcase name="verifyLoginTest" time="10.85" classname="com.boss.qa.tests.LoginPageTest"/>
</testsuite> <!-- login page test -->
